ceno_zkvm/src/tables/program.rs

```diff
@@ -172,7 +172,17 @@ impl<E: ExtensionField, const PROGRAM_SIZE: usize> TableCircuit<E>
                 }
             });
 
-        Self::padding_zero(&mut fixed, num_fixed).expect("padding error");
+        assert_eq!(INVALID as u64, 0, "cannot use 0 as program padding");
```

Comment:
Sorry, why is 0 not allowed?

Reply:
Yes that would work too. The spirit is to have a generic padding strategy, treating all fields at once, not aware of type InsnKind. Ultimately I’d like to refactor it into such generic `padding` function.

Reply:
The comment could be "0 must be valid padding".